src/extension-runners/firefox-desktop.js

```diff
@@ -217,10 +217,9 @@ export class FirefoxDesktopExtensionRunner {
       startUrl,
       firefoxApp,
       firefoxClient,
+      binaryArgs = [],
     } = this.params;
 
-    const binaryArgs = [];
-
     if (browserConsole) {
       binaryArgs.push('-jsconsole');
```

Comment:
uhm... I'm thinking that it may be better to append the additional arguments that the user may be passing to a `web-ext run` command (instead of prepending them before the ones added internally by web-ext itself).

@Rob--W what do you think about it?

Reply:
Appending rather than prepending sounds good to me.